operations/observability/Observability.md

```diff
@@ -0,0 +1,365 @@
+- product status (status is marked as succeeded if all the stages' statuses are succeeded, otherwise it is marked as failed)
+
+### Timing dashboard
+
+- Dynamic top-down and bottom-up dashboard to track install/upgrade status of any product, stage, and operation.
+- Dashboard calculates the execution time for the install/upgrade of any product, stage, and operation.
+- The status of the install/upgrade for product, stage, and operation are failed or succeeded.
+- There is a dropdown for selection of the product, stage, and operation. By default all are selected.
+  The framework defines and implements periodic scheduled health checks to occur both during and after installation or upgrade.
+  The frequency can be determined by administrator by running the tests over a period of time and selecting the appropriate time interval in the Grafana dashboard to check the results for past six hours, a day, and a week.
+- Separate sections are created in the dashboard to see the details of product, stage, and operation.
+- There is a graph showing the duration for each stage and operation is added.
+
+![IUF timing dashboard](../../img/operations/TimingDashboard.png "Timing Dashboard")
+
+## Goss tests for PIT and NCN
+
+### Goss test overview
+
+The observability tooling monitors the logs for Goss tests run by using the automated scripts. These scripts are regularly run during install/upgrade. For each Goss test, metrics are generated on its duration and success or failure.
+
+This framework provides a set of quantifiable metrics used to create a visual Grafana health dashboard of all environments. This provides administrators with insight on which tests fail most, or which clusters have the most problems.
+
+Trend analysis of this data in Grafana across different dimensions may point out statistically where the highest frequency of issues occurs.
+It also provides administrators an at-a-glance dashboard where they can visually see the complete system health. This includes the option to drill down from the aggregate view of the environment to the functional areas or the components.
+
+Goss test logs are scraped using grok-exporter and visualized on Grafana using captured data.
+[`csm-testing` repository](https://github.com/Cray-HPE/csm-testing/tree/main/goss-testing/) contains all of the tests, suites, and scripts to execute these Goss tests.
+Automated scripts run the Goss tests in batches. These batches are based on functionality or the check performed in CSM.
+
+A single Goss test is a YAML file and a collection of these Goss tests can be used together by adding them in another YAML file, called a test suite. These suites can be invoked through the CSM-provided scripts to get a log file with results for the tests.
+
+For example, a Goss test to validate expected Kubernetes nodes exists using the `kubectl` command based on node names
+pulled from `/etc/hosts` file. The log files generated from automated scripts that run a set of test suites are used.
```

Comment:
This example seems like it is more likely to be confusing to people than to be helpful. I'm not sure what information it is providing that is relevant to the observability framework page. I'd suggest removing it.

```suggestion
```